.github/workflows/ci.yml

```
@@ -5,19 +5,15 @@ jobs:
   golangci:
     strategy:
       matrix:
-        go-version: [1.16.x]
         platform: [ubuntu-latest]
     name: golangci-lint
     runs-on: ${{ matrix.platform }}
     steps:
       - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
         with:
-          fetch-depth: 1
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{ matrix.go-version }}
-      - run: |
-          docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.37.1 golangci-lint run -v
+          version: v1.43.0
```

```suggestion
          version: v1.37.1
```
let's not break things in this PR